docs/workbench/HowToRun.md

```diff
@@ -13,67 +13,145 @@
    limitations under the License.
 -->
 # How To Run Submarine Workbench
+We provide two methods to launch Submarine Workbench
+*  Method 1:  Run Submarine Workbench on docker
+*  Method 2:  Run Submarine Workbench without docker
 
-## Run Submarine on docker
+# Run Submarine Workbench on docker
 
-By using the official image of submarine, only one docker command is required to run submarine workbench.
+By using the official images of Submarine, only a few docker commands are required to launch **Submarine Workbench**. The document includes information about how to launch the Submarine Workbench via the new docker images and the information about how to switch between different Submarine Workbench versions(version Vue & version Angular). 
 
-It should be noted that since the submarine workbench depends on the submarine database, so you need to run the docker container of the submarine database first.
+### Two versions of Submarine Workbench
+1. Angular (default)
+2. Vue (This is the old version, and it will be replaced by version Angular in the future.)
+#### (WARNING: Please restart a new **incognito window** when you switch to different versions of Submarine Workbench) 
+### Launch the Submarine Workbench(Angular)
+* It should be noted that since Submarine Workbench depends on the Submarine database, so you need to run the docker container of the Submarine database first.
+```
+docker run -it -p 3306:3306 -d --name submarine-database -e MYSQL_ROOT_PASSWORD=password apache/submarine:database-0.3.0-SNAPSHOT
+docker run -it -p 8080:8080 -d --link=submarine-database:submarine-database --name submarine-server apache/submarine:server-0.3.0-SNAPSHOT
+```
+* The login page of Submarine Workbench will be shown in ```127.0.0.1:8080```.
```

Comment:
Please change `127.0.0.1:8080` to `http://127.0.0.1:8080`.

Reply:
Thanks for your review. I have fixed it.